docs/core_docs/src/theme/Feedback.js

```diff
@@ -161,6 +164,33 @@ export default function Feedback() {
     setFeedbackSent(true);
   };
 
+  const handleFeedbackDetails = async (e) => {
+    e.preventDefault();
+    if (!feedbackId) {
+      setFeedbackDetailsSent(true);
```

Comment:
if `feedbackDetailsSent` is true do we want to return or allow the user to submit multiple times?

Also I don't understand why we're returning here & setting true if there is no ID?

Reply:
If there's no id there, you're in a bad state and best thing to do is mark as done and no-op